src/app/ubs/ubs-user/ubs-user-profile-page/ubs-profile-delete-pop-up/ubs-profile-delete-pop-up.component.scss

```diff
@@ -1,9 +1,56 @@
 .dialog-container {
-  width: 450px;
-  padding: 20px;
-  font-family: var(--secondary-font);
-}
+  width: 400px;
+  padding: 20px 30px 40px 30px;
+  font-family: var(--primary-font);
+
+  .title {
+    font-family: var(--primary-font);
+    font-weight: 700;
+    font-style: normal;
+    font-size: 20px;
+    color: var(--ubs-primary-black);
+    line-height: 28px;
+    letter-spacing: 0.002em;
+  }
+
+  .message {
+    font-family: var(--primary-font);
+    font-weight: 400;
+    font-style: normal;
+    font-size: 16px;
+    color: var(--ubs-primary-grey);
+    line-height: 24px;
+  }
+
+  .delete,
+  .cancel {
+    font-family: var(--primary-font);
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.1px;
+    color: var(--ubs-quaternary-dark-grey);
+    padding: 8px 24px;
+    width: 123px;
+    height: 36px;
+    border: none;
+  }
+
+  .delete {
+    background-color: var(--ubs-button-light-green);
+  }
+
+  .delete:hover {
+    background-color: var(--ubs-button-dark-green);
+  }
```

Comment:
SASS nested selector could be used: &:hover

Reply:
Good point. Thanks.